webapp/src/components/common/dialogs/DigestDialog/DigestMatrix.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2024, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import { MatrixGridSynthesis } from "@/components/common/Matrix/components/MatrixGridSynthesis";
+import {
+  generateDataColumns,
+  generateResultColumns,
+  groupResultColumns,
+} from "@/components/common/Matrix/shared/utils";
+import { Box } from "@mui/material";
+import type { DigestMatrix } from "./types";
+import EmptyView from "../../page/SimpleContent";
+import { GridOff } from "@mui/icons-material";
+import { useTranslation } from "react-i18next";
+
+const isGroupedColumns = (
+  columns: string[] | string[][],
+): columns is string[][] => {
+  return Array.isArray(columns[0]);
+};
+
+const isColumns = (columns: string[] | string[][]): columns is string[] => {
```

Comment:
Opinion: Not necessary to create this one, it's more clear to use `isGroupedColumns` very time for me